src/components/sound-card/sound-card.jsx

```diff
@@ -18,6 +18,7 @@ export function SoundCard({ soundInfo }) {
 
   return (
     <img
+      data-test-id="soundcard-audio-img"
```

Comment:
could you try to add title to the id? Something like this
```suggestion
      data-test-id={`soundcard-audio-img${card.title ? "-card.title" : ""}`}
```